Tools/auterion/remote_update_fmu.sh

```diff
@@ -50,53 +50,66 @@ while getopts ":f:c:d:p:u:" opt; do
 				exit 1
 			fi
 			;;
+		r )
+			revert=true
+			;;
 	esac
 done
 
-target_dir=/shared_container_dir/fmu
+if [ -z "$device" ]; then
+	echo "Error: missing device"
+	exit 1
+fi
 
-# create update-dev.tar
-target_file_name="update-dev.tar"
-tmp_dir="$(mktemp -d)"
-config_path=""
-firmware_path=""
+if [ "$revert" == true ]; then
+	# revert to the release version which was originally deployed
+	cmd="cp /persistent/shared_container_dir/fmu/update.tar /persistent/shared_container_dir/fmu/update-dev.tar"
```

Comment:
Can you use `target_dir=/shared_container_dir/fmu` from below?

Reply:
How's the mounting? Does it also work with `/shared_container_dir/fmu` instead of `/persistent/shared_container_dir/fmu`. Looking at the SCP command it should but I haven't checked 🤔

Reply:
I reused the variable like in the scp command for flashing without the persistent root folder in https://github.com/PX4/PX4-Autopilot/commit/cdc784eeb3b5257fcb6f38406705224748b88dbd. Echoing the output it's now:
`cp /shared_container_dir/fmu/update.tar /shared_container_dir/fmu/update-dev.tar`
but I can't test since I'm missing the hardware now. Could someone give this command a try? 😇